site/docs/v1/tech/apis/scim/SCIMGroup.adoc

```diff
@@ -0,0 +1,159 @@
+---
+layout: doc
+title: SCIM Group APIs
+description: APIs for provisioning SCIM Groups in FusionAuth using your SCIM Client
+---
+
+== Overview
+
+This page contains all of the APIs for managing Groups through SCIM Group requests.
+
+* <<Create a Group>>
+* <<Delete a Group>>
+* <<Retrieve a Group>>
+* <<Update a Group>>
+
+== Create a Group
+
+This API is used to create a new FusionAuth Group from a SCIM request.
+
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Create a Group from a SCIM request
+[.endpoint]
+.URI
+--
+[method]#POST# [uri]#/api/scim/resources/v2/Groups#
+--
+
+:http_method: POST
+include::/docs/v1/tech/apis/scim/_scim-group-request-body.adoc[]
+
+=== Response
+
+The response for this API contains the Group that was just created in SCIM schema format.
+
+:webhook_event:
+:create_user:
+include::/docs/v1/tech/apis/_standard-post-response-codes.adoc[]
+:create_user!:
+:webhook_event!:
+
+For FusionAuth SCIM endpoints, any error responses will be returned in standard SCIM schema. See more details in the link:/docs/v1/tech/apis/scim[SCIM API Overview].
```

Comment:
Is this worth breaking out to an include? I see it often :)

Reply:
You mean just that one sentence? That seems like maybe overkill, but I can do it if you think it's useful.